build.gradle

```diff
@@ -44,7 +44,7 @@ subprojects {
     apply plugin: 'org.inferred.processors'
 
     tasks.check.dependsOn(javadoc)
-    sourceCompatibility = 1.8
+    sourceCompatibility = 11
```

Comment:
We don't support 1.8 anymore, this is reasonable. Still worth classifying the PR as a `break`.